protocol/communities/community.go

```diff
@@ -1451,6 +1451,25 @@ func (o *Community) TokenPermissionsByType(permissionType protobuf.CommunityToke
 	return permissions
 }
 
+func (o *Community) ChannelTokenPermissionsByType(channelID string, permissionType protobuf.CommunityTokenPermission_Type) []*protobuf.CommunityTokenPermission {
+	permissions := make([]*protobuf.CommunityTokenPermission, 0)
+	for _, tokenPermission := range o.TokenPermissions() {
+		if tokenPermission.Type == permissionType && includes(tokenPermission.ChatIds, channelID) {
+			permissions = append(permissions, tokenPermission)
+		}
+	}
+	return permissions
+}
+
+func includes(channelIDs []string, channelID string) bool {
```

Comment:
I think it would be nice to add this fn as a lambda inside ChannelTokenPermissionsByType fn?

Reply:
Hm... that'd be redefining the function for every call though.. 
I can do it if there's strong opinions

Reply:
No strong opinion whichever way